### ISSUE TITLE
Not working for me

### ISSUE THREAD

**Issue by Obsidian2004** [SteamAchievementNotifier/SteamAchievementNotifier#10]
Hey guys first I love that this exist! I been looking for something like this for at least 5years.
That being said the program doesn't seem to be working for me...  So at first it was detecting the games giving me the launch with game notification but no achievement sound when they popped. So I figured I missed something and I did I had my game details in private. So changed it to public. Now the launch notification isn't showing up and sound still isn't working... 
finally I played with the settings nothing. re-re confirmed my account is linked, it is. If I minimize the game I can see that the notifier has detected the game every-time. Last I reseted settings tried again no launch notification. So finally I uninstalled and reinstalled still nothing. So I forced my self to sign up on git-hub and now I'm here. Hopefully I can get some help.

**Comment by Obsidian2004** [comment]
After a little more messing around I'm noticing a few indie games are not launching with the notifier launch tag.
So far : Salt and sanctuary no notifier on launch 
            Darksouls3 Launches with notifier 
            Jotun Valhalla edition no notifier on launch
            Monster Hunter Rise with notifier
            Loop Hero no notifier on launch
I need to test if the sound works with the ones that launch with the notifier but those larger games are not easy.

**Comment by SteamAchievementNotifier** [comment]
Hey @Obsidian2004 - sorry to hear you're having issues!

All Steam games should be tracked by the app (even if they have no achievements), so it seems like it may not have installed correctly.

Initially, make sure that the file is not blocked:

> - Close Steam Achievement Notifier.
> - Right-click "SteamAchievementNotifierV1.72.exe" > _Properties_.
> - Just above the "OK"/"Cancel"/"Apply" buttons in the _Properties_ menu, if you see a checkbox saying "Unblock", check it and click "Apply".
> - Restart and see if your games are tracked. If not, try to uninstall again after unblocking and reopen the app.

Also, where is your EXE file located? It would be easiest to run it from somewhere easily accessible, such as your Downloads folder, but if it is a "special folder" (such as _Program Files (x86)_), Windows sometimes has difficulty setting a path to locations like that.

Feel free to [join the Discord server](https://discord.gg/FxCFtpd3eu) if you prefer!

**Comment by Obsidian2004** [comment]
Sorry for the long wait I can only get on at late night. kids....
So I went into properties but I don't have a unblock button to check. I have hidden/read only/advanced option nothing in advanced either.
Figured I should try other things while I was in there and ran it under administrator just to try something no luck...

It was under my Downloads folder but I deleted it again because I felt the in app reset uninstall wasn't actually uninstalling. For example the short cut stayed on my start menu and there wasn't any prompt to run the program last time. So I reset uninstalled the app and deleted shortcut and executable until I was sure it was scrubbed. It does not seem to show up in the uninstall apps and programs list. I re-downloaded it and installed it in my music folder just to see if anything would change nothing. Not sure where to go from here. Thanks for getting back to me though.

**Comment by Obsidian2004** [comment]
OK just because I'm ocd and like testing stuff I went and deleted it again. So I moved the EXE. to the desktop to avoid any folder path and ran the installation as administrator just to see.. and got a lot of nothing new again... New PC too figures... Might need to get on my other PC and do a discord after all... -.- Ill check here in the morning. PM a time I think Ill have some free time tomorrow.

**Comment by SteamAchievementNotifier** [comment]
Hey again @Obsidian2004.

The Downloads folder should work fine, along with Desktop.

Also, the app is portable so it won't show up in Control Panel's uninstall menu, as nothing is actually "installed", so clicking "RESET" > "Uninstall" in the app should delete everything generated by Steam Achievement Notifier on the system.

Was the remaining Start Menu shortcut active? (i.e. when clicking on it, did it open the program, or did it show the "Windows cannot find this program" dialog?)

The only thing I can think of right now is that your environmental variables in Windows are set weirdly somehow - if you open Command Prompt, type `echo %userprofile%` and press Enter, what does it say your user profile path is? (This would usually be something like `C:\Users\username`.)

Also, could you check whether you are getting any errors within the app? You can press `CTRL + Shift + i` when the app window is open to bring up the console, which should give a bit more information on what is wrong. (FYI - if you want to post a screenshot, make sure to hide any references to your "API Key" in the console/app window.)

Thanks!

**Comment by Obsidian2004** [comment]
Hey!
So yea that start menu shortcut was still active if I uninstalled threw the app reset. It would pop the app back up with the shortcut after, just without any info on it so no reinstall prompt. if I deleted EXE then the shortcut would give me the no app found prompt.
With command looks normal C:\Users\Myusername
CTRL + Shift + i is just taking screen shots of whatever I have on screen?  Not getting extra info or error log while the app is on the screen.

**Comment by Obsidian2004** [comment]
SO to sum all this up! If anyone runs into this problem just change privacy on your steam profile for the sound and if you don't get the steam achievement notifier launch prompt in some games, but its fine in others during startup its because some games run in full screen and some in border-less full screen. I thought the app was still malfunction because most the games I tested were indie games, while bigger developers tend to use full screen border-less more(The more you know). It really seems to depend on the developer of the game. I did run into a few indies that used border-less and the prompt appeared. Thanks to my man up top for meeting me on discord and helping me out I'm a big fan of this app!
